cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -87,6 +88,8 @@ public abstract class AbstractConnectOperator<C extends KubernetesClient, T exte
     private static final ReconciliationLogger LOGGER = ReconciliationLogger.create(AbstractConnectOperator.class.getName());
 
     private final boolean isNetworkPolicyGeneration;
+    private final FeatureGates featureGates;
```

Comment:
Maybe you can keep only boolean and not the whole feature gates class? Something like `boolean continueOnManualRUFauilure`. Again, this might be good in all 3 classes.